crates/generator/src/traits.rs

```diff
@@ -171,10 +175,13 @@ impl<S: State + CodeStore> StateExt for S {
                 addr
             }
         };
-        self.mint_sudt(CKB_SUDT_ACCOUNT_ID, &address, capacity.into())?;
+        // Align CKB to 18 decimals
+        let ckb_amount = ckb_decimal::to_18(capacity);
+        self.mint_sudt(CKB_SUDT_ACCOUNT_ID, &address, ckb_amount)?;
         log::debug!(
-            "[generator] mint {} shannons CKB to account {}",
-            capacity,
+            "[generator] mint {} shannons * 10^{} CKB to account {}",
+            ckb_amount,
```

Comment:
```suggestion
capacity
```

Reply:
@zeroqn Please correct this log message.